API.adoc

```diff
@@ -0,0 +1,402 @@
+=== SMART COSMOS Things Edge API Documentation
+
+The SMART COSMOS Things Edge API provides a standard set of CRUD methods for
+creating,reading, updating, and deleting a Thing and its associated Metadata
+in a single operation. Each CRUD action requires that the User executing the
+call has at least one Role with the appropriate Authority or Authorities. The
+Authorities relevant to the Things Edge API are listed below.
+
+== Authorities
+
+* https://authorities.smartcosmos.net/metadata/create
+* https://authorities.smartcosmos.net/metadata/update
+* https://authorities.smartcosmos.net/metadata/read
+* https://authorities.smartcosmos.net/metadata/delete
```

Comment:
That's because api.smartcosmos.net hasn't been updated with my last week changes
to smartcosmos-webpages-authorities. As soon as that happens, all the underlying
text is right.